docs/sources/reference/loki-http-api.md

```diff
@@ -264,13 +264,17 @@ curl -H "Content-Type: application/json" \
 ## Ingest logs using OTLP
 
 ```bash
-POST /otlp/v1/logs
+POST /otlp
```

Comment:
same here I belive. The true route is `/otlp/v1/logs` its just the otel collector adds the last bit for you